<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>My Work</title>
  <link rel="stylesheet" href="style.css">
</head>
<body>
  <header>
    <nav>
      <a href="index.html">Home</a>
      <a href="biology.html">Biology</a>
      <a href="drones.html">Drones</a>
      <a href="glassblowing.html">Glassblowing</a>
      <a href="reading.html">Reading</a>
    </nav>
  </header>

  <main>
    <h1>My Research</h1>


    <h2>Chen Lab</h2>
    <p class="subheading">Broad Institute, 2023-present</p>
    <p>
      I'm currently a 3rd-year PhD student in Professor Fei Chen's <a href="https://www.insitubiology.org/">lab</a> where I aim to develop 0-to-1 technologies for biology.
      I have two main projects:</p>
      
      <ul>
        <li>
        I'm developing a way to deliver therapeutic proteins specifically to the Golgi and ER using retrograde trafficking.
        I'm very excited about this project because it blends protein engineering with cell biology.
        </li>
        <li>
          I'm developing a single-cell proteomics method that has high specificity and is compatible with single-cell RNA sequencing.
          Ultimately, we hope to simultaneously measure cells' entire proteomes and transcriptomes to study mRNA translationa and identify therapeutic opportunities in cancer.
      </li>
      
      </ul>
    <p>
      I also helped enable the spatial transcriptomics technology <a href="https://www.nature.com/articles/s41586-023-06837-4">Slide-Tags</a> be able to capture single-cell TCR and BCR sequences.
      The technology is being used to study immune responses in clinical samples, like melanoma and lymphoma (see <a href="https://www.nature.com/articles/s41467-025-62709-7">here</a>).
    </p>
    Below is a cool picture I took of the Golgi body in a primary mouse fibroblast
      
    <figure>
      <img src="images/biology/PMF.png" alt="Golgi of mouse fibrblast" width="600">
      <figcaption>Staining of GOLIM4 in a primary mouse fibroblast</figcaption>
    </figure>
    </p>

    <h2>Baker Lab</h2>
    <p class="subheading">University of Washington, Summer 2022</p>
    <p>
      I had the opportunity to design proteins with deep learning in Professor David Baker's Lab (pre-Nobel!) and was fortunate to be mentored by <a href="https://www.linkedin.com/in/dougtischer/">Dr. Doug Tischer</a>.
      My goal was to design a minibinder to FGFR using only deep learning-based protein design <a href="https://www.nature.com/articles/s41586-021-04184-w">tools</a>, instead of physics-based ones, for generating sequences.
      Although my designs had strong <i>in silico</i> AlphaFold2 metrics, none of them bound when I tested them experimentally!
      Below are the 4 classes of designs I came up with - it is cool to see the interfaces protein hallucination came up with!
    <figure>
      <img src="images/biology/binders.jpg" alt="My binder designs" width="600">
      <figcaption>The minibinders I designed against FGFR (blue)</figcaption>
    </figure>
    </p>

    <h2>Langer Lab</h2>
    <p class="subheading">MIT, 2020-2022</p>
    <p>
      I joined the Langer Lab during my freshman year because I was interested in nanomedicine and had the privilege of being mentored by <a href="https://www.garywliu.com/pi"> Dr. Gary Liu.
      For my first project, we set out to develop a treatment for uremia using a polymer-protected urease.
      To assess our idea's feasibility, I used Python to model the treatment, and the results showed the approach was not feasible which saved months of lab work.
      It was cool to see how mathematical modeling can effectively guide experimental work.
    </p>
    <p>
      XXXX
    </p>
      
    <figure>
      <img src="images/biology/binders.jpg" alt="My binder designs" width="600">
      <figcaption>The minibinders I designed against FGFR (blue)</figcaption>
    </figure>
    </p>
  </main>
</body>
</html>