---
layout: default
title: Research
---
<script defer src="https://unpkg.com/3dmol/build/3Dmol-min.js"></script>

<h1>Research</h1>

<h2>Chen Lab</h2>
<p class="subheading">Broad Institute, 2023-present</p>
<p>
  I'm currently a 3rd-year PhD student in Professor Fei Chen's <a href="https://www.insitubiology.org/">lab</a> where I aim to develop 0-to-1 technologies for biology.
  I have two main projects:</p>
  
  <ul>
    <li>
    I'm developing a way to deliver therapeutic proteins specifically to the Golgi and ER using retrograde trafficking.
    I'm very excited about this project because it blends protein engineering with cell biology.
    </li>
    <li>
      I'm developing a single-cell proteomics method that has high specificity and is compatible with single-cell RNA sequencing.
      Ultimately, we hope to simultaneously measure cells' entire proteomes and transcriptomes to study mRNA translation and identify therapeutic opportunities in cancer.
  </li>
  
  </ul>
<p>
  I also helped enable the spatial transcriptomics technology <a href="https://www.nature.com/articles/s41586-023-06837-4">Slide-Tags</a> be able to capture single-cell TCR and BCR sequences.
  The technology is being used to study immune responses in clinical samples, like melanoma and lymphoma (see <a href="https://www.nature.com/articles/s41467-025-62709-7">here</a>).
</p>
<p>
  Below is a cool picture I took of the Golgi body in a primary mouse fibroblast
</p>

<figure>
  <img src="images/research/PMF.png" alt="Golgi of mouse fibrblast" width="600">
  <figcaption>Staining of GOLIM4 in a primary mouse fibroblast</figcaption>
</figure>

<h2>Baker Lab</h2>
<p class="subheading">University of Washington, Summer 2022</p>
<p>
  I had the opportunity to design proteins with deep learning in Professor David Baker's Lab (pre-Nobel!) and was fortunate to be mentored by <a href="https://www.linkedin.com/in/dougtischer/">Dr. Doug Tischer</a>.
  My goal was to design a minibinder to FGFR using only deep learning-based protein design <a href="https://www.nature.com/articles/s41586-021-04184-w">tools</a>, instead of physics-based ones, for generating sequences.
  Although my designs had strong <i>in silico</i> AlphaFold2 metrics, none of them bound when I tested them experimentally!
  Below are the 4 classes of designs I came up with - it is cool to see the interfaces protein hallucination came up with!
</p>

<figure>
  <img src="images/research/binders.jpg" alt="My binder designs" width="600">
  <figcaption>The minibinders I designed against FGFR (blue)</figcaption>
</figure>

<h2>Langer Lab</h2>
<p class="subheading">MIT, 2020-2022</p>
<p>
  To pursue my interest in nanomedicine, I joined the Langer Lab during my freshman and had the privilege of being mentored by <a href="https://www.garywliu.com/pi"> Dr. Gary Liu</a>.
  For my first project, we set out to develop a treatment for uremia using a polymer-protected urease.
  To assess our idea's feasibility, I used Python to model the treatment, and the results showed the approach was not feasible which saved months of lab work.
  It was cool to see how mathematical modeling can effectively guide experimental work. If you are interest, you can find a brief writeup of my model <a href="research/model.pdf" target="_blank" rel="noopener noreferrer">here</a>.
</p>
<p>
  As I took more biology classes, I became more interested in molecular biology.
  We came up with the idea of using gene editing to install a mutation in the cell surface protein CD59 to reduce rates of cardiovascular disease in diabetes patients. Below is an (interactive) model of the CD59 protein I was trying to engineer.
  Unfortunately, we could not install the mutation in cells, but I learned many fundamental biology techniques, and more importantly, how to formulate questions and design experiments to answer them.
  I also became interest in protein engineering and design, which led me to the Baker Lab in the Summer of 2022.
</p>
  
<div id="pdb-viewer" style="width:200px; height:200px; position: relative; margin: 0 auto;">
</div>

<script>
  const element = document.getElementById('pdb-viewer');
  const config = { backgroundColor: 'white' };
  const viewer = $3Dmol.createViewer(element, config);

  const pdbUrl = 'https://files.rcsb.org/download/2J8B.pdb';
  fetch(pdbUrl)
      .then(r => r.text())
      .then(pdbText => {
      viewer.addModel(pdbText, 'pdb');
      viewer.setStyle({}, { cartoon: { color: 'spectrum' } });
      viewer.zoomTo();
      viewer.render();
    });

  window.addEventListener('resize', () => viewer.resize());
</script>

<h3>Aerospace Controls Lab</h3>
<p class="subheading">MIT, 2019</p>
<p>
  When I got to MIT, I wanted to do research in drones, so I began working at the Aerospace Controls Lab where I was mentored by <a href="https://andrewtorgesen.com/">Andrew Torgesen</a>.
  The goal of our project was to use a drone to measure air flow data behind a ship to improve ship design (see <a href="https://acl.mit.edu/projects/ship-aerowake-measurement-system">here</a>).
  Through this experience, I was introduced to ROS and the many components that go into drones. Below is a video of our drone in the testing facility.
  This project was a great introductory experience to drone research, and although I switched to biological research afterwards, I continued to build and read about FPV drones as a hobby.
</p>

<video controls preload="metadata" height="400px" width="100%">
  <source src="images/research/drone.mov" type="video/mp4">
</video>